### ISSUE TITLE
(Newbie) Made Tux (Linux Mascot) in Openmoji Style. (Feedback pls)

### ISSUE THREAD

**Issue by andrei-akopian** [hfg-gmuend/openmoji#471]
I made [Tux](https://en.wikipedia.org/wiki/Tux_(mascot)) the mascot of the Linux Operating System in Openmoji style.
I am new to vector art so feel free to critique, in fact, I am looking for feedback.
(Optimized Export... should have removed the template)
![tux-openmoji-v001](https://github.com/hfg-gmuend/openmoji/assets/86117540/5693c6d7-44b0-4a18-bed5-bb5c35e25fa2)
(Raw file)
![tux-openmoji](https://github.com/hfg-gmuend/openmoji/assets/86117540/738c485b-e663-415b-8d9b-14aa4e38e1cc)

**Comment by b-g** [comment]
Hi @andrei-akopian, many thanks for this! Having Tux in the set would be super! 

Feedback:

The beak of the penguin looks a bit sad because the corners of the mouth are pointing downward. Here are a few variations of the beak for inspiration ... I would definitely make Tux a bit happier :)

![IMG_7419](https://github.com/hfg-gmuend/openmoji/assets/480224/261fae0b-7ea7-4163-bca1-825cb908ee4c)


I think the eyes are a bit too big. Maybe they could be made slightly smaller. Another idea would be to use small arcs instead of the current circles. That would also make it look a lot happier ... I guess something in between would be the best solution.

![IMG_7415](https://github.com/hfg-gmuend/openmoji/assets/480224/f65cd2f5-f1f2-4b1e-8b93-c135c6810234)


And last please don't forget to check you svg files with https://openmoji-tester.glitch.me/, currently the overall shape is missing a white background fill, see the 2nd test.

**Comment by sotha1** [comment]
hello, i am a linux enthusiast and love openmoji project, so i tried make tux

obs: i never used inkscape before, so i'm sorry
![tux](https://github.com/hfg-gmuend/openmoji/assets/149703967/ac190f9a-33b2-45bb-886c-10a0de6b8901)

**Comment by dnlutz** [comment]
@sotha1 thank you for your svg. i made some minor tweaks: symetrical body, uniform foot colour. is that o.k. ?
![231108_linux_tux](https://github.com/hfg-gmuend/openmoji/assets/38292227/85cf4818-69e5-40e4-97fd-c3803a77b4ca)

**Comment by andrei-akopian** [comment]
Personal opinion, your Tux is a bit to vertical. I think he should have a bit more width to him. I think the proportions should be the way I made it, but the full lines and coloring do look better. Again, I am not an expert.

**Comment by sotha1** [comment]
> @sotha1 thank you for your svg. i made some minor tweaks: symetrical body, uniform foot colour. is that o.k. ? ![231108_linux_tux](https://user-images.githubusercontent.com/38292227/281385393-85cf4818-69e5-40e4-97fd-c3803a77b4ca.svg)

yep!

**Comment by sotha1** [comment]
> Personal opinion, your Tux is a bit to vertical. I think he should have a bit more width to him. I think the proportions should be the way I made it, but the full lines and coloring do look better. Again, I am not an expert.

i agree to you, the final version should be more widther. but i don't know if dnlutz will tweak this too

**Comment by danik292** [comment]
Heeya I agree
